.run-gha-tests.sh

```diff
@@ -71,12 +72,13 @@ case "$1" in
         opam exec -- dune exec -- obuilder healthcheck --store=rsync:/rsync
         opam exec -- dune exec -- ./stress/stress.exe --store=rsync:/rsync
 
-        # Populate the caches from our own GitHub Actions cache
-        sudo mkdir -p /rsync/cache/c-opam-archives
-        sudo cp -r ~/.opam/download-cache/* /rsync/cache/c-opam-archives/
-        sudo chown -R 1000:1000 /rsync/cache/c-opam-archives
-
-        opam exec -- dune exec -- obuilder build -f example.spec . --store=rsync:/rsync
+        # Rsync tests are slow (17 mins), disable them for now.
+        # # Populate the caches from our own GitHub Actions cache
+        # sudo mkdir -p /rsync/cache/c-opam-archives
+        # sudo cp -r ~/.opam/download-cache/* /rsync/cache/c-opam-archives/
+        # sudo chown -R 1000:1000 /rsync/cache/c-opam-archives
+        #
+        # opam exec -- dune exec -- obuilder build -f example.spec . --store=rsync:/rsync
```

Comment:
This should be restored when merging to `master` we want to be testing the rsync backend.